CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ list(REMOVE_ITEM files
   ${FOLLY_DIR}/experimental/io/HugePageUtil.cpp
   ${FOLLY_DIR}/python/fibers.cpp
   ${FOLLY_DIR}/python/GILAwareManualExecutor.cpp
+  ${FOLLY_DIR}/folly/executor.cpp
```

Comment:
Will this ever match anything?  Assuming the cython build generates executor.cpp, won't this end up in the build directory as `cybld/folly/executor/.cpp` rather than directly in `folly/`?